notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb

```diff
@@ -18,6 +18,8 @@
     "- [Original Pipeline Inference](#FastSAM-in-Ultralytics)\n",
```

Comment:
<p>As I understand, after your changes the interactive demo will still use the original model and not the quantized one. </p><p> </p><p>Usually, when quantization is added, the interactive demo is also changed to visually demonstrate the comparison between the models. You can refer to PR #1411 for an example of such demo changes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>I added a radio button allowing to switch between the quantized and regular models</p>

Reply:
<p>Thanks! Could you please also add an additional output for the time it took to make prediction? Otherwise benefits of quantization are not reflected in the demo.</p>

Reply:
<p>Also, seems like radio button needs to be hidden in case quantization is not enabled. Otherwise fp32 model is executed nevertheless of radio button value.</p>

Reply:
> Also, seems like radio button needs to be hidden in case quantization is not enabled. Otherwise fp32 model is executed nevertheless of radio button value.

I see it as over-complication. What if the user has the flag unchecked but the quantized checkpoint is present? And what if the flag is checked but the checkpoint is not there? I think it is good enough right now. @eaidova what do you think?

Reply:
> > Thanks! Could you please also add an additional output for the time it took to make prediction? Otherwise benefits of quantization are not reflected in the demo.
> 
> Gradio indicates the latest inference time

Yes, I've seen it in some cases. But here I don't see it:
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/23343961/6693d881-2dd2-422b-9a67-7f083a76a85b)

Reply:
In fact I don't see any difference in time execution in gradio (about 1-2 seconds). Looks like gradio brings a lot of overhead costs and we can only demonstrate that we have the same output for both models.

Reply:
I see. Then indeed no need to add a time output.

Reply:
> > Also, seems like radio button needs to be hidden in case quantization is not enabled. Otherwise fp32 model is executed nevertheless of radio button value.
> 
> I see it as over-complication. What if the user has the flag unchecked but the quantized checkpoint is present? And what if the flag is checked but the checkpoint is not there? I think it is good enough right now. @eaidova what do you think?

These are the valid questions.

> What if the user has the flag unchecked but the quantized checkpoint is present?

Then we don't load the quantized model and show it in demo.

> And what if the flag is checked but the checkpoint is not there?

This should error out, as it currently will. Because `quantized_wrapped_model` either will not be declared or would not be able to be loaded.

> I think it is good enough right now. 

Sorry, I disagree. When I select quantized model, I expect the quantized model to be executed, not the original one.

I suggest the following change:

```
model_type = gr.Radio(
            ["FP32 model", "Quantized model"] if do_quantize.value else ["FP32 model"],
            value="FP32 model", label="Select model variant"
        )
```
I don't think it makes it much more complicated compared to how it is now.

Reply:
Added proposed code block